### PR TITLE
JSON file generator tests

### PR DESCRIPTION
**Pull request by alongreyber**

### CODE REVIEW

File: tests/test_generator.py [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
+    ]
 
-def test_boolean():
-    """Test generating booleans."""
-    schema = {
-        "type": "boolean"
-    }
-    output = generate_json(schema)
-    assert isinstance(output, bool)
+    for value in generated_json_values:
```
Comment (patrickkwang) [inline tests/test_generator.py:36]: We can make this more concise and more efficient in failure cases by combining the validation loop and the generation list comprehension.

**Review by patrickkwang — CHANGES_REQUESTED** [review]

**Comment by patrickkwang** [comment]
Since our tests are actually stochastic, it may not be simple to reproduce a failed test (due to a real bug). Do we get enough information from pytest to at least recall the specific instance that failed validation?

**Comment by alongreyber** [comment]
That is a good point. How about we add `random.seed(0)` to the top of the test file? I just tried it and it produces the same values every time.

**Comment by patrickkwang** [comment]
That's an option. I'm not opposed to the tests being irreproducible _if_ we get enough information to identify the offending instance.

**Comment by alongreyber** [comment]
Should be fixed now. Validation errors will now log the instance that was being validated.

**Review by patrickkwang — APPROVED** [review]

**Pull request merged**